### PR TITLE
feat(consolidator): consolidator user testing feedback

### PR DESCRIPTION
**Pull request by HejdaJakub**
* Two cards were merget to on with white backgroud.
* Logo in header now redirects to home. In the header is now also logout button and the button with url to another application (user-profile).
* Linked accounts are now in rows, with email in brackets (if is available) and with remove button. This button opens already existing dialog.
* When linker is closed and result is OK, then we newly do not redirect user to result page, but we just reload home page.

### CODE REVIEW

File: apps/consolidator/src/assets/i18n/en.json [hunk 1]
```diff
@@ -32,6 +36,25 @@
...
+          "SUBJECT_ERROR": "Subject cannot be empty.",
+          "MESSAGE": "Message",
+          "MESSAGE_PLACEHOLDER": "Please describe the issue",
+          "MESSAGE_HINT": "Please try to describe in detail the problem.",
+          "CANCEL": "Cancel",
+          "SEND": "Send",
+          "SUCCESS": "Issue report was sent and got ticket number: "
+        },
+        "REMOVE_USER_EXT_SOURCE": {
+          "TITLE": "Remove user's account",
```
Comment (xkureck) [inline apps/consolidator/src/assets/i18n/en.json:52]:
```suggestion
          "TITLE": "Remove account",
```

**Review by xkureck — CHANGES_REQUESTED** [review]
image

Also is the BREAKING CHANGE necessary? All the added config properties are not breaking anything if they stay on their default values. A dunno, Just asking

File: apps/consolidator/src/app/components/connect-identity-section/connect-identity-section.component.ts [hunk 2]
```diff
@@ -44,7 +44,9 @@ export class ConnectIdentitySectionComponent implements OnInit {
     this.openLinkerService.openLinkerWindow((result: LinkerResult) => {
       if (result === 'TOKEN_EXPIRED') {
         location.reload();
-      } else if (result === 'OK' || result === 'MESSAGE_SENT_TO_SUPPORT') {
+      } else if (result === 'OK') {
+        window.location.reload();
```
[the remark this answers is not in the document]
Comment (HejdaJakub) [reply apps/consolidator/src/app/components/connect-identity-section/connect-identity-section.component.ts]: Refactored - connect-identity component merged with main-window component, so now is easier to update data after some changes from linker

**Comment by HejdaJakub** [comment]
> Also is the BREAKING CHANGE necessary? All the added config properties are not breaking anything if they stay on their default values. A dunno, Just asking
You are right, it's not probably necessary to mark it as a BREAKING CHANGE.

**Review by xkureck — APPROVED** [review]

**Pull request merged**